{"code":"ENOTFOUND","errno":-3008,"hostname":"postgres","level":"error","message":"Database connection error: getaddrinfo ENOTFOUND postgres","stack":"Error: getaddrinfo ENOTFOUND postgres\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:13:5)","syscall":"getaddrinfo"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"postgres","level":"error","message":"Database connection error: getaddrinfo ENOTFOUND postgres","stack":"Error: getaddrinfo ENOTFOUND postgres\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:13:5)","syscall":"getaddrinfo"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"postgres","level":"error","message":"Database connection error: getaddrinfo ENOTFOUND postgres","stack":"Error: getaddrinfo ENOTFOUND postgres\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:13:5)","syscall":"getaddrinfo"}
{"code":"42601","file":"scan.l","length":93,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \"user\"","name":"error","position":"44","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \"user\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42601","file":"scan.l","length":93,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \"user\"","name":"error","position":"44","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \"user\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42601","file":"scan.l","length":97,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \"MODIFY\"","name":"error","position":"5068","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \"MODIFY\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"23502","column":"sector","file":"tablecmds.c","length":146,"level":"error","line":"6051","message":"Database connection error: column \"sector\" of relation \"project\" contains null values","name":"error","routine":"ATRewriteTable","schema":"public","severity":"ERROR","stack":"error: column \"sector\" of relation \"project\" contains null values\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)","table":"project"}
{"code":"42601","file":"scan.l","length":97,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \"CREATE\"","name":"error","position":"5364","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \"CREATE\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42701","file":"tablecmds.c","length":144,"level":"error","line":"7249","message":"Database connection error: column \"disbursement\" of relation \"fundingsource\" already exists","name":"error","routine":"check_for_column_name_collision","severity":"ERROR","stack":"error: column \"disbursement\" of relation \"fundingsource\" already exists\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42701","file":"tablecmds.c","length":144,"level":"error","line":"7249","message":"Database connection error: column \"disbursement\" of relation \"fundingsource\" already exists","name":"error","routine":"check_for_column_name_collision","severity":"ERROR","stack":"error: column \"disbursement\" of relation \"fundingsource\" already exists\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42701","file":"tablecmds.c","length":144,"level":"error","line":"7249","message":"Database connection error: column \"disbursement\" of relation \"fundingsource\" already exists","name":"error","routine":"check_for_column_name_collision","severity":"ERROR","stack":"error: column \"disbursement\" of relation \"fundingsource\" already exists\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42701","file":"tablecmds.c","length":144,"level":"error","line":"7249","message":"Database connection error: column \"disbursement\" of relation \"fundingsource\" already exists","name":"error","routine":"check_for_column_name_collision","severity":"ERROR","stack":"error: column \"disbursement\" of relation \"fundingsource\" already exists\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42601","file":"scan.l","length":97,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \"CREATE\"","name":"error","position":"5414","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \"CREATE\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42601","file":"scan.l","length":92,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \";\"","name":"error","position":"2067","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \";\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"42601","file":"scan.l","length":92,"level":"error","line":"1245","message":"Database connection error: syntax error at or near \";\"","name":"error","position":"2067","routine":"scanner_yyerror","severity":"ERROR","stack":"error: syntax error at or near \";\"\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:16:5)"}
{"code":"28000","length":37,"level":"error","message":"Database connection error: SSL/TLS required","name":"error","severity":"FATAL","stack":"error: SSL/TLS required\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:13:5)"}
{"code":"ECONNRESET","errno":-4077,"level":"error","message":"Database connection error: read ECONNRESET","stack":"Error: read ECONNRESET\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:13:5)","syscall":"read"}
{"code":"28000","length":37,"level":"error","message":"Database connection error: SSL/TLS required","name":"error","severity":"FATAL","stack":"error: SSL/TLS required\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:13:5)"}
{"code":"ECONNRESET","errno":-4077,"level":"error","message":"Database connection error: read ECONNRESET","stack":"Error: read ECONNRESET\n    at C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\shahabuddin akhon hr\\Desktop\\climate\\backend\\src\\config\\db.js:14:5)","syscall":"read"}
{"code":"ECONNREFUSED","level":"error","message":"Database connection error:","stack":"AggregateError [ECONNREFUSED]: \n    at C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\src\\config\\db.js:16:5)"}
{"code":"ECONNREFUSED","level":"error","message":"Database connection error:","stack":"AggregateError [ECONNREFUSED]: \n    at C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\src\\config\\db.js:16:5)"}
{"code":"ECONNREFUSED","level":"error","message":"Database connection error:","stack":"AggregateError [ECONNREFUSED]: \n    at C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\src\\config\\db.js:16:5)"}
{"code":"ECONNREFUSED","level":"error","message":"Database connection error:","stack":"AggregateError [ECONNREFUSED]: \n    at C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\src\\config\\db.js:16:5)"}
{"code":"ECONNREFUSED","level":"error","message":"Database connection error:","stack":"AggregateError [ECONNREFUSED]: \n    at C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\src\\config\\db.js:16:5)"}
{"code":"ECONNREFUSED","level":"error","message":"Database connection error:","stack":"AggregateError [ECONNREFUSED]: \n    at C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async connectDB (C:\\Users\\maish\\OneDrive\\Desktop\\Coding\\climate-finance\\backend\\src\\config\\db.js:16:5)"}
